crs_motion_planning/src/path_processing_utils.cpp

```diff
@@ -70,35 +70,6 @@ bool crs_motion_planning::parsePathFromFile(const std::string& yaml_filepath,
   return true;
 }
 
-void crs_motion_planning::tesseractRosutilsToMsg(trajectory_msgs::msg::JointTrajectory& traj_msg,
```

Comment:
It looks like you don't need tesseract here either anymore. You should be able to remove `  tesseract::tesseract_common` from the target_link_libraries